src/js/views/dashboards/dashboard_views.js

```diff
@@ -24,8 +25,21 @@ const ContextTrailView = View.extend({
 });
 
 const IframeView = View.extend({
-  className: 'flex-grow',
-  template: hbs`<iframe src="{{ embed_url }}"></iframe>`,
+  className: 'flex-grow quicksight-dashboard',
+  template: hbs`<div class="js-dashboard-container"></div>`,
+  ui: {
+    dashboardContainer: '.js-dashboard-container',
+  },
+  onRender() {
+    QuicksightEmbedding.createEmbeddingContext().then((embeddingContext => {
+      embeddingContext.embedDashboard({
+        url: this.model.get('embed_url'),
+        container: this.ui.dashboardContainer,
+        height: '100%',
+        width: '100%',
+      });
+    }));
+  },
```

Comment:
Not really sure what `createEmbeddingContext()` does, but it needs to be initiated first before the dashboard embed can happen...